src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_PASSWORD_LENGTH = 6;
+    private static final int MIN_AGE_ALLOWED = 18;
+    private static final String PASSWORD = "123456";
+    private RegistrationService service;
+    private StorageDao storageDao;
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        service = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        user = new User();
+    }
+
+    @AfterEach
+    void tearDown() {
+        service = null;
+        storageDao = null;
+        user = null;
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        assertThrows(RuntimeException.class, () -> service.register(null));
+    }
+
+    @Test
+    void register_nullUserLogin_NotOk() {
+        user.setLogin(null);
```

Comment:
how can we be sure that we get this exception because on null login, not null age? All other fields except login for user should be valid (same in other tests)